hack/remove-calico-policy/override-policy.md

```diff
@@ -10,7 +10,8 @@ ingress and egress traffic, overriding any other Network Policy that has been co
 
 - Calico version v2.3 or higher
 - Kubernetes v1.6 or higher
-- These steps assume Calico is running in policy-only mode (without Calico networking)
+- Calico is running in policy-only mode (without Calico networking)
+- Calico is using using the Kubernetes API as the datastore
```

Comment:
I don't think we should use kubernetes backend in the docs and if we do then we are mixing a bunch of different terms.

As a descriptive statement we use the term "Kubernetes API as the datastore" in the main doc:

http://docs.projectcalico.org/v2.2/getting-started/kubernetes/installation/hosted/kubernetes-datastore/